v20.2/performance-benchmarking-with-tpc-c-10k-warehouses.md

```diff
@@ -144,14 +150,14 @@ You'll be importing a large TPC-C data set. To speed that up, you can temporaril
     $ cockroach sql --insecure --host=<address of any node>
     ~~~
 
-3. Disable replication:
+3. Disable replication: (<strong><font style="size:xx-large" color="red">FIXME</font></strong>: are these still the right cluster settings?)
```

Comment:
For the 13k page I'm not sure what to edit.  I'm guessing for starters one no longer disables replication (as for 140k).

It might (?) be easier if there is a gdoc somewhere describing the "diff" for the 13k run similar to what exists for the 140k.  Do we have something like that?